tsconfig.json

```diff
@@ -7,13 +7,14 @@
     "allowSyntheticDefaultImports": true,
     "strict": true,
     "forceConsistentCasingInFileNames": true,
-    "module": "commonjs",
+    "module": "esnext",
```

Comment:
why ?

Reply:
when i specify ` "module": "commonjs" `, i get the warning:
```
(!) Plugin typescript: @rollup/plugin-typescript: Rollup requires that TypeScript produces ES Modules. Unfortunately your configuration specifies a "module" other than "esnext". Unless you know what you're doing, please change "module" to "esnext" in the target tsconfig.json file or plugin options.
```

I will try to change it in the plugin options

Reply:
I see. Then it's fine, keep what you did. Can resolve the issue once you read that message

Reply:
you changed in the end ?

Reply:
yes, changed
```
typescript({ module: "esnext" }),
```